spec/zipkin/tracer_spec.rb

```diff
@@ -67,6 +67,105 @@
     end
   end
 
+  describe '#start_active_span' do
+    let(:operation_name) { 'operator-name' }
+
+    context 'when a root span' do
+      let(:scope) { tracer.start_active_span(operation_name) }
+      let(:span) { scope.span }
+
+      describe 'span context' do
+        it 'has span_id' do
+          expect(span.context.span_id).not_to be_nil
+        end
+
+        it 'has trace_id' do
+          expect(span.context.trace_id).not_to be_nil
+        end
+
+        it 'does not have parent_id' do
+          expect(span.context.parent_id).to be_nil
+        end
+      end
+    end
+
+    context 'when a child span context' do
+      let(:root_span) { tracer.start_span(root_operation_name) }
+      let(:scope) { tracer.start_active_span(operation_name, child_of: root_span.context) }
+      let(:span) { scope.span }
+      let(:root_operation_name) { 'root-operation-name' }
+
+      describe 'span context' do
+        it 'has span_id' do
+          expect(span.context.span_id).not_to be_nil
+        end
+
+        it 'has trace_id' do
+          expect(span.context.trace_id).not_to be_nil
+        end
+
+        it 'does not have parent_id' do
+          expect(span.context.parent_id).not_to be_nil
+        end
+      end
+    end
+
+    context 'when a child span' do
+      let(:root_span) { tracer.start_span(root_operation_name) }
+      let(:scope) { tracer.start_active_span(operation_name, child_of: root_span) }
+      let(:span) { scope.span }
+      let(:root_operation_name) { 'root-operation-name' }
+
+      describe 'span context' do
+        it 'has span_id' do
+          expect(span.context.span_id).not_to be_nil
+        end
+
+        it 'has trace_id' do
+          expect(span.context.trace_id).not_to be_nil
+        end
+
+        it 'does not have parent_id' do
+          expect(span.context.parent_id).not_to be_nil
+        end
+      end
+    end
+
+    context 'when already existing active span' do
+      let(:root_operation_name) { 'root-operation-name' }
+
+      it 'uses active span as a parent span' do
+        tracer.start_active_span(root_operation_name) do |parent_scope|
+          tracer.start_active_span(operation_name) do |scope|
+            expect(scope.span.context.parent_id).to eq(parent_scope.span.context.span_id)
+          end
+        end
+      end
+    end
+  end
+
+  describe '#active_span' do
+    let(:root_operation_name) { 'root-operation-name' }
+    let(:operation_name) { 'operation-name' }
+
+    it 'returns current active span' do
```

Comment:
"returns currents active span" doesn't seem to adequately describe the behavior being tested. Can this be more descriptive?